modules/openapi-generator/src/main/resources/kotlin-client/build.gradle.mustache

```diff
@@ -98,6 +101,10 @@ dependencies {
     compile "io.reactivex.rxjava2:rxjava:$rxJava2Version"
     compile "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
     {{/useRxJava2}}
+    {{#useRxJava3}}
+    compile "io.reactivex.rxjava3:rxjava:$rxJava3Version"
+    compile "com.github.akarnokd:rxjava3-retrofit-adapter:3.0.0"
```

Comment:
What if we use the `com.squareup.retrofit2:adapter-rxjava3` RxJava3 adapter from square?

Reply:
@4brunu When I run `./bin/generate-samples.sh bin/configs/kotlin*`now, the samples will delete all rxjava3 code lines, I don't know why.